spec/spec_helper.rb

```diff
@@ -0,0 +1,21 @@
+require 'webmock/rspec'
+require 'keyrod'
+
+WebMock.disable_net_connect!(allow_localhost: true)
+
+RSpec.configure do |config|
+  config.color = true
+  config.tty = true
+  config.order = 'random'
+
+  config.before do
+    stub_request(:post, 'https://aai-dev.egi.eu/oidc/token?client_id=keyrod&client_secret=3HcNV88ex48UKYPWCI1OwresZAxMjh1Pdf&grant_type=refresh_token&refresh_token=iOiI0ZDQ5MzUxMC00MDJhLTQ3NDQtYjE4Yi&scope=openid%20profile')
```

Comment:
You should do the stubbing in the test class you are actually testing not in the general configuration.